pkgs/development/libraries/rapidjson/default.nix

```diff
@@ -13,14 +13,16 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ pkgconfig cmake ];
 
-  # detected by gcc7
-  NIX_CFLAGS_COMPILE = [ "-Wno-error=implicit-fallthrough" ];
+  preConfigure = ''
+    substituteInPlace CMakeLists.txt --replace "-Werror" ""
```

Comment:
Can you add a comment why we need to disable errors completely?

Reply:
We could add
```-Wno-error=zero-as-null-pointer-constant``` and ```-Wno-error=shadow``` to use with modern clang, but what is the point? Compilers will get more sophisticated diagnostics, there will be more warnings and it's not like we actually going to fix any of the code. So we should just let developers compile with ```-Werror```